lib/utils/udpsender.js

```diff
@@ -0,0 +1,20 @@
+require('./KalturaConfig');
+
+var PORT = KalturaConfig.config.udpsender.PORT;
+var HOST = KalturaConfig.config.udpsender.HOST;
+var client = require('dgram').createSocket('udp4');
+
+var sendFunction = function (message,response){
+    var message_buffer = new Buffer(message);
+    response.log('#UDP ' + message_buffer + ' sent to ' + HOST + ':' + PORT);
+    client.send(message_buffer, 0, message_buffer.length,PORT, HOST,
+        function (err, bytes) {

```

Comment:
Note on UDP datagram size:  - the smaller the datagram the higher chance in will delivered successfully.  I think DNS protocol restrict its message to 512 Bytes.  Another thing to node is that if the receiver MTU is smaller than the datagram length you will end up with fragmentation. This is important since UDP isn't reliable protocol - so let's make sure we are passing only what's really needed